README.md

```diff
@@ -1,2 +1,16 @@
 # conda-metadata-app
-A streamlit app to query metadata from conda packages
+A streamlit app to query metadata from conda packages.
+
+> [!TIP]
+> A public instance of this app is available at [**conda-metadata-app.streamlit.app**](https://conda-metadata-app.streamlit.app).
+
+![Main Screenshot](docs/assets/main_screenshot.png)
```

Comment:
Bonus points for light and dark mode 😜


```suggestion
![Main Screenshot](docs/assets/main_screenshot_dark.png#gh-dark-mode-only)
![Main Screenshot](docs/assets/main_screenshot_light.png#gh-light-mode-only)
```


https://github.com/pavelzw/pixi-pycharm/blob/6c855ce94b5a776dfeb59329904b1cff85ac6faf/README.md?plain=1#L55

Reply:
Collected :D
48db171c6cb26b385fdd21454247a0ff6e783b29